xbmc/ServiceManager.cpp

```diff
@@ -207,6 +209,7 @@ void CServiceManager::DeinitStageOne()
   m_XBPython.reset();
 #endif
   m_announcementManager.reset();
+  m_network.reset();
```

Comment:
Can you place this at the beginning of the function to make sure the destruction order is opposite initialization?

Reply:
while the previous code shows that there is no order issues, it's good to be consistent. changed.